Wikipedia/Code/wikipedia-language-variants.json

```diff
@@ -0,0 +1,66 @@
+{
```

Comment:
I think it's appropriate that this is manual (given how rarely these are updated), but we should probably update [this document](https://github.com/wikimedia/wikipedia-ios/blob/main/docs/languages.md) to mention that language variants are manually updated here.

Reply:
Updated the document with details.